backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -191,9 +191,16 @@
       end
 
       context 'splitting to location' do
-        # can not properly implement until poltergeist supports checking alert text
-        # see https://github.com/teampoltergeist/poltergeist/pull/516
-        it 'should warn you if you have not selected a location or shipment'
+        it 'should warn you if you have not selected a location or shipment' do
+          visit spree.edit_admin_order_path(order)
+
+          within('tr', text: line_item.sku) { click_icon 'arrows-h' }
+          click_icon :ok
+          alert = page.driver.browser.switch_to.alert
+
+          expect(alert.text).to eq 'Please select the split destination.'
+          alert.accept
```

Comment:
I think you can remove lines 199, 201, 202, and replace it with `accept_confirm { Please select the split destination }`

Reply:
Thanks for pointing this out @aitbw! 

Following [docs](https://github.com/teamcapybara/capybara/blob/0fe411a6dadf5028c12560ce77b43358ad1a019f/lib/capybara/session.rb#L610-L630) I've done this way at the end:

```ruby
accept_alert 'Please select the split destination.' do
  click_icon :ok
end
```